test/e2e_node/jenkins/OWNERS

```diff
@@ -6,3 +6,5 @@ approvers:
 - Random-Liu
 - yguo0905
 - yujuhong
+emeritus_approvers:
+- krzyzacy
```

Comment:
nit: new line, and... this file... 🙈 I bet nobody is active in this list...

Reply:
I've got some suggestions for this file but that doesn't need to happen as part of this PR

Reply:
right, I'd like to add people, but I want to discuss with them first and make that a different PR later, I also think we should probably put sig testing in more directories, this alias doesn't really reflect things the SIG reviews currently.

Reply:
shouldn't you actually remove krzyzacy from the approvers list in this file at the same time?

Reply:
pffft as-if /s

fixed.